python-frontend/src/main/java/org/sonar/python/types/v2/FunctionType.java

```diff
@@ -135,4 +138,8 @@ public PythonType owner() {
   public TypeOrigin typeOrigin() {
     return typeOrigin;
   }
+
+  public String fullyQualifiedName() {
```

Comment:
Since now we have let's say two types having `fqn` (the `UnresolvedImportType` and the `FunctionType` and in nearest future we may want to add it to more types - I think it makes sense to introduce an interface like:
```java
public interface HasFullyQualifiedName {
  String fullyQualifiedName();
}
```

Reply:
As mentioned in the previous comment, `UnresolvedImportType` does not have a `fqn`.

We could have another interface that returns `importPath` (which would be the FQN for `FunctionType` and `ClassType`). I'm not yet sure about that though, as I'm afraid it could be misused.